pg_migration_tool/main.py

```diff
@@ -96,6 +96,7 @@ async def check_connection_for_db(self, db, label) -> bool:
                 user=db["db_username"],
                 password=db_password,
                 host=db["db_connection_host"],
+                port=db.get("port", 5432),
```

Comment:
db_connection_port

Reply:
you frigging kidding me

Reply:
that is a database section, what other kind of port can be?

Reply:
`db_` is redundant everywhere, we should probably drop it completely from the DB section

Reply:
ok